apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SessionPoolManager.cs

```diff
@@ -161,12 +161,12 @@ internal void Release(SessionPool sessionPool)
         internal IReadOnlyList<Statistics> GetStatistics() =>
             _targetedPools.ToArray().Select(tp => tp.Value.GetStatisticsSnapshot()).ToList().AsReadOnly();
 
-        internal SessionPool.DatabaseStatistics GetDatabaseStatistics(SpannerClientCreationOptions options, DatabaseName databaseName)
+        internal SessionPool.DatabaseStatistics GetDatabaseStatistics(SpannerClientCreationOptions options, DatabaseName databaseName, string databaseRole)
```

Comment:
The fact that all these methods are changing in the same way strongly suggests to me that we want a new type to represent "the key for a targeted pool within a SessionPool".

The naming of that could be interesting - we have TargetedSessionPool at the moment, but that's internal.

I suggest you change this PR to create such a type (currently with just a database name and creator role) and we can worry about the naming afterwards.

Reply:
I now see it already exists as SessionPoolInfo - I'll add more comments there, but basically I think it should be made public and exposed in the APIs.